WindowsInstaller/config_template.txt

```diff
@@ -81,108 +40,298 @@ allow_user_actions_while_paused = false-----------------------------------------
+-- Allow user actions while game is paused
+-- In Theme Hospital the player would only be allowed to use the top menu if
+-- the game was paused. That is the default setting in CorsixTH too, but by
+-- setting this to true everything is allowed while the game is paused.
+--
+allow_user_actions_while_paused = false
+
+---------------------------------------------------------------------------------------------------
+-- VOLUME CONTROL IS OPENING THE DRUG CASEBOOK?
+
+-- If your keyboard volume control opens the Drug Casebook at the same time
+-- then change this to true.  From then on you will have to use Shift + C to open
+-- the Casebook and volume down will not open it.
+-- For example for shift + C to open casebook change the setting below to = true
+--
+volume_opens_casebook = false
+
+---------------------------------------------------------------------------------------------------
+-- To allow patients with Alien DNA to visit your hospital other than by an emergency change
+-- the settings below.  Understand that there are no animations for sitting down, opening
+-- or knocking on doors etc.
+-- So, like with Theme Hospital to do these things they will appear to change to normal
+-- looking and then change back.
+--
+alien_dna_only_by_emergency = true
+alien_dna_must_stand = true
+alien_dna_can_knock_on_doors = false
+
+-- To allow female patients with fractured bones, which are by default disabled due to poor
+-- animation that skips and jumps a bit
+--
+disable_fractured_bones_females = true
+
+---------------------------------------------------------------------------------------------------
+-- By default the player selects any extra objects they want for each room they build.
+-- If you would like the game to remember what you usually add, then change this option to true.
+--
+enable_avg_contents = false
+---------------------------------------------------------------------------------------------------
+-- By default destroyed rooms can't be removed. If you would like the game to
+-- give you the option of removing a destroyed room change this option to true.
+--
+remove_destroyed_rooms = false
+
+----------------------------------------------- FOLDER SETTINGS -----------------------------------
+-- These settings can also be changed from the Folders Menu
+---------------------------------------------------------------------------------------------------
+---------------------------------------------------------------------------------------------------
+-- Theme hospital install folder: original game data files are loaded from this
+-- folder. Between the square braces should be the folder which contains the
+-- original HOSPITAL.EXE and/or HOSP95.EXE file. This can point to a copy of
+-- the Theme Hospital demo, though a full install of the original game is
+-- preferred.
+--
+theme_hospital_install = [[/Users/tobylane/.config/CorsixTH/Theme Hospital]]
```

Comment:
Most users are Windows for this config no? Probably best to use the Windows format.

``C:\Program Files (x86)\Theme Hospital``
^ other variants are available, annoyingly. And assuming it let you install the game on Windows. Otherwise use the GOG default I guess of ``C:\Program Files (x86)\GOG.com\Theme Hospital``

Reply:
This file is automatically generated from config_finder.lua ... actually it is curious that toby's name ended up in it.